### ISSUE TITLE
Setting currentTime unexpectedly triggers play for YouTube and Vimeo

### ISSUE THREAD

**Issue by gurupras** [sampotts/plyr#876]
### Expected behaviour
Setting `currentTime` directly or through `forward()` should only seek the video and not change the state of the player
### Actual behaviour
On a video that has never been played, `plyr.currentTime = 30` will seek 30 seconds in and start playing the video. On a paused video, this will only seek
### Environment
- Browser: Chrome/Firefox
- Version:
  - Chrome: 65.0.3325.181
  - Firefox: 59.0.1
- Operating System: Windows 10
- Version: Build 1703

### Steps to reproduce
Here's a [pen](https://codepen.io/gurupras/pen/LdgeeO)

**Comment by gurupras** [comment]
This does seem to be the default HTML5 video behavior. Unfortunately, attempting to pause on `seeked` doesn't seem to work

```
plyr.on('seeked', function _once () {
  plyr.off('seeked', _once)
  if (expectedState === 'paused') {
    plyr.pause()
  }
})
```

**Comment by gurupras** [comment]
Yeah, you're right. I think it may have been some listener I had active without realizing. I tried reproducing this on the [w3org-mediaevents](https://www.w3.org/2010/05/video/mediaevents.html) page and was unsuccessful.

The core problem pointed out in this issue still holds though.  
Should I just handle this manually rather than expect `plyr` to do it for me? I need to seek to a specific part of the video while also ensuring that the video state is respected. To do this, I'm currently having to do the following:
```
plyr.on('canplay', function () {
  if (startTime !== 0) {
    // Changing currentTime seems to implicitly trigger play
    plyr.on('seeked', function _once () {
      plyr.off('seeked', _once)
      if (expectedState === 'paused') {
        // Directly calling plyr.pause() does not work for some reason
        setTimeout(() => { plyr.pause() }, 10)
      }
      plyr.currentTime = startTime
    }
  })
})
```

**Comment by sampotts** [comment]
I can't reproduce this on either Chrome or Firefox as stated.

As a side note there's a `playing` getter on the `Plyr`.

**Comment by sampotts** [comment]
OK just seen it's with YouTube.

**Comment by gurupras** [comment]
I thought I did follow the issue template, but yes, I should've made it a lot more clearer that the issue was with youtube/vimeo and not native HTML videos. The fact that I had an unreproducible follow-up comment  only made it worse. Sorry about that.

**Comment by sampotts** [comment]
Seems to also effect Vimeo. I'll look into it. It's a bit of a pain they do that.

**Comment by friday** [comment]
Youtube seems to set the state to `video cued` (5) by default until you play. If you play and pause it will become `paused` (2), but it doesn't seem like you can pause it directly from `video cued` (5).

The reason this matters is youtube's [`seekTo()`](https://developers.google.com/youtube/iframe_api_reference#seekTo) starts playing unless the state is `paused` (2)

**Comment by friday** [comment]
Reopening since it seems like I reintroduced this in #1021, but only when ads are enabled (like on the demo).

Too many moving parts for my sanity... 😞

**Comment by AirJulien** [comment]
Hi everyone,

I had this problem too after putting back the `currentTime` at a specific time when the `ended` event was triggered.

The most simple solution for me was to catch the `play` event one time :

```
plyr.on("ended", () => {
    plyr.once("playing", () => {
        plyr.pause();
    });
    plyr.currentTime = ...
});
```
